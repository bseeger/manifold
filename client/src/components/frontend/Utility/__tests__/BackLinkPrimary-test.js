import React from 'react';
import renderer from 'react-test-renderer';
import BackLinkPrimary from '../BackLinkPrimary';

describe("Frontend.Utility.BackLinkPrimary component", () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <BackLinkPrimary
        link="test/link"
        title="test"
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
