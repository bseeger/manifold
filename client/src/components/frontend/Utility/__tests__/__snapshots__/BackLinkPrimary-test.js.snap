exports[`Frontend.Utility.BackLinkPrimary component renders correctly 1`] = `
<a
  className="back-link-primary full"
  onClick={[Function]}
  style={Object {}}>
  <div
    className="container flush">
    <i
      className="manicon manicon-arrow-left" />
    Back to Project
    <span>
      test
    </span>
  </div>
</a>
`;
